samigo/samigo-app/src/java/org/sakaiproject/tool/assessment/ui/listener/delivery/DeliveryActionListener.java

```diff
@@ -579,17 +581,13 @@ else if (action == DeliveryBean.TAKE_ASSESSMENT_VIA_URL) {
 
   }
   
-  protected void registerIrss(DeliveryBean delivery, Event event, boolean isViaURL) {
-	  LearningResourceStoreService lrss = (LearningResourceStoreService) ComponentManager
-			  .get("org.sakaiproject.event.api.LearningResourceStoreService");
-	  if (null != lrss && lrss.getEventActor(event) != null) {
-		  StringBuffer lrssMetaInfo = new StringBuffer("Assesment: " + delivery.getAssessmentTitle());
-		  lrssMetaInfo.append(", Past Due?: " + delivery.getPastDue());
-		  if (isViaURL) {
-			  lrssMetaInfo.append(", Assesment taken via URL.");
-		  }
-		  lrss.registerStatement(getStatementForTakeAssessment(lrss.getEventActor(event), event, lrssMetaInfo.toString()), "samigo");
+  protected LRS_Statement registerIrss(DeliveryBean delivery, boolean isViaURL) {
```

Comment:
How much of a PITA would it be to change it to be correct, since LRSS (Learning Record Store Service) is the proper acronym? Just my OCD kicking in here...